modules/systemd.nix

```
@@ -105,6 +105,10 @@ in {
 
   options = {
     systemd.user = {
+      enable = mkEnableOption "The user systemd service manager" // {
+        default = pkgs.stdenv.isLinux;
```

To avoid documentation dependency on pkgs:

```suggestion
        default = pkgs.stdenv.isLinux;
        defaultText = literalExpression "pkgs.stdenv.isLinux";
```